crates/storage/src/libmdbx.rs

```diff
@@ -216,4 +290,74 @@ mod tests {
         };
         assert_eq!(read_value, Some(value));
     }
+
+    #[test]
+    fn store_and_fetch_block() {
```

Comment:
Does it make sense to move tests to `crate/storage/src/lib` just as [test_store_account](https://github.com/lambdaclass/ethereum_rust/pull/146/files#diff-ca59a3bc257083e85b6a9c0d7d761d2c7538f96a196c6d2fe1d891dbb6965bb0R232)?
That way we'll have tests for all Engine implementations right away